code/modules/clothing/glasses/thermal.dm

```diff
@@ -54,3 +54,11 @@
 	icon_state = "thermalimplants"
 	item_state = "syringe_kit"
 	toggleable = 0
+
+/obj/item/clothing/glasses/thermal/m64_thermal_goggles
```

Comment:
This is pretty powerful and a balance concern.

Reply:
You won't get night vision from this, and most maps have a lot of glass windows or are wide open anyways so thermal vision would only work in some areas.